Chromium/Makefile

```diff
@@ -1,9 +1,6 @@
 all: tables guide content dist
 
-# Do not remove exporting of SHARED variable!!! (since
-# shared/transforms/combineremediations.py helper
-# requires its value to be available)
-export SHARED = ../shared
```

Comment:
AFAIK the "introduction" of passing ```SHARED``` variable between the child and parent Makefiles was necessary before to pass the location of ```shared/``` dir from product Makefile to the global Makefile (to subsequently pass this location to the XSLT).

But it seems there's different way how to pass this information (without exporting ```SHARED```), and it works (since Jenkins test succeeded). So exporting ```SHARED``` can be removed.